packages/utils/src/link-header.ts

```diff
@@ -2,7 +2,7 @@ import LinkHeader from 'http-link-header';
 
 export function getApplicationRegistrationIri(linkHeaderText: string): string {
   const links = LinkHeader.parse(linkHeaderText).refs;
-  return links.find((link) => link.rel === 'http://www.w3.org/ns/solid/interop#registeredApplication')?.anchor;
+  return links.find((link) => link.rel === 'http://www.w3.org/ns/solid/interop#registeredAgent')?.anchor;
```

Comment:
@elf-pavlik can you double check whether this should be `Agent` or `Application` or both? In the service side of things it is checking for `Agent` https://github.com/janeirodigital/sai-impl-service/blob/3b0def205c234f355daee36f619ae2fbc74a7304/packages/service/src/handlers/agents-handler.ts#L42

Reply:
it should be Agent